react/react1/week2/todo-list/src/components/Timer/Timer.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from "react";
+import { useState } from "react";
+
+export const Timer = () => {
+  const [count, setCount] = useState(() => {
+    console.log("Timer started");
+    return 0;
+  });
+
+  useEffect(() => {
+    setTimeout(() => {
+      setCount((prevCount) => prevCount + 1);
+    }, 1000);
+  });
```

Comment:
You mixed up setTimeout with setInterval :) 
setTimeout calls a function only **once** after 1000ms, while setInterval is the one that calls it **every** 1000ms
Additionally, you should remember to clean up intervals or timers you're setting inside the useEffect hook.
You do that in a return statement! [Hint](https://devtrium.com/posts/set-interval-react)